config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/matt/opentelemetry-operator/opentelemetry-operator
-  newTag: 0.82.0-9-g6f4246c
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

Comment:
should we remove this?

Reply:
yes, also the `hcr.io/matt/opentelemetry-operator/opentelemetry-operator` should be removed it got accidentially merged in  https://github.com/open-telemetry/opentelemetry-operator/pull/1980